etc/db_schema.xml

```diff
@@ -10,6 +10,9 @@
         <column xsi:type="varchar" name="po_number" nullable="false" default="" length="255" comment="PO Number"/>
         <column xsi:type="varchar" name="order_request_id" nullable="false" default="" length="255" comment="Punchout Request Id"/>
         <column xsi:type="text" name="extra_data" nullable="true" comment="Punchout Extra Data"/>
+		<constraint xsi:type="primary" referenceId="PRIMARY">
```

Comment:
Is this just GH not formatting it correctly?  Or is not aligned with the other nodes?